pkg/deployment/cert.go

```diff
@@ -116,8 +116,9 @@ func EnsureTLSCerts(ctx context.Context, helper *helper.Helper,
 			issuerLabelSelector = map[string]string{service.Spec.TLSCert.Issuer: ""}
 		}
 
-		issuer, err = certmanager.GetIssuerByLabels(ctx, helper, "", issuerLabelSelector)
+		issuer, err = certmanager.GetIssuerByLabels(ctx, helper, instance.Namespace, issuerLabelSelector)
```

Comment:
this is currently in the tls-e defaulting patch but it makes more sense to include that here.
for now I'm going to rebase my pr on this pr so we can get a ci run without waiting for this to merge.

we can merge this patch indepently however.